### ISSUE TITLE
Water types needs balancing

### ISSUE THREAD

**Issue by PingPongLee** [Glitchfiend/ToughAsNails#332]
I've played for a while with this mod now, and one thing stands out that needs balancing: The water types. 

There simply is no good reason to spend time filtering, boiling and creating juice. It ofcourse helps much more on thirst, but compared to just using a canteen to drink dirty water it is not worth it. If i simply keep close to a water source there's no reason to create juice.

To create three juices, which instead of 1,5 thirst "points" gives you 4 (which is only around 2,6 times better) you need: 3 wood blocks, 6 sugarcanes (3 for filters and 3 for sugar) and 3 of the different fruits/vegetables you want your juice in. I think that's quite expensive for something that is not even 3 times better than just using the dirty water which only costs you a canteen once.

I know a rain collector is giving you clean water, and that's great, but I still believe the filtering/boiling way should be made more relevant. 

Suggestion: Either make the dirty water actually hurt the player when drinking it, or make the charcoal filters last longer than one use.

**Comment by Forstride** [comment]
I've changed the thirst values around a little bit.

- Dirty water now only restores 1 thirst point (Formerly 1.5), and adds 5% hydration (Formerly 10%).  It also now has a 100% chance of giving the Thirst status effect (Formerly 80%).

- Filtered water now restores 2 thirst points (Formerly 2.5).  It also now has a 50% chance of giving the thirst effect (Formerly 40%).

- And clean water now restores 3 thirst points (Formerly 3.5).

So clean water is now 3x better than dirty water, and dirty water is much more dangerous to drink, especially since I've fixed the Thirst status effect.  Additionally, I'll look into making it so charcoal filters can be used multiple times (Maybe 3 times each?)
